src/Symfony/Bridge/Monolog/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+5.4.0
```

Comment:
5.3

Reply:
I though it was too late for 5.3 (stabilization phase). But ok :)